src/ray/util/stream_redirection_utils.cc

```diff
@@ -22,77 +22,35 @@
 #include <vector>
```

Comment:
I'd also rename this file to just `stream_redirection`, but up to you

Reply:
no strong opinion here, updated